drv/gimlet-hf-server/src/main.rs

```diff
@@ -16,6 +16,7 @@
         target_board = "gimlet-c",
         target_board = "gimlet-d",
         target_board = "gimlet-e",
+        target_board = "gimlet-f",
     ),
     path = "bsp/gimlet_bcde.rs"
```

Comment:
Want to rename this to `gimlet_bcdef.rs`?

Reply:
Agh, sorry!